### DISCUSSION TITLE
Transition value & animation on swipe up/down

### DISCUSSION THREAD

**Discussion by MarianPliatsko** [LucasMucGH/BottomSheet#145] [Q&A]
Hello!  I need to make bottom sheet to follow the finger without animation (bottom sheet should follow the finger without delay). I was trying to use customAnimation(nil) but then everything is just freezing. I will be very pleased for your help. Thank you!

@lucaszischka

**Comment by lucaszischka** [comment]
To my understanding of your problem, you can just use `.linear()` instead of `nil` as you would with normal SwiftUI.

**Reply by MarianPliatsko** [reply]
Is it possible to configure it as I described below?

**Comment by MarianPliatsko** [comment]
@lucaszischka  

The bottom sheet does not follow the finger but slows down, so there is an ever-widening gap between the position of the finger and that of the bottom sheet.

A have this:

https://github.com/lucaszischka/BottomSheet/assets/103671984/5792725f-b7f0-492f-b68e-45c7ef3dcccb

And trying to get this:

https://github.com/lucaszischka/BottomSheet/assets/103671984/2b2ff241-2d0b-4a31-af48-206b7a161b6b

**Reply by lucaszischka** [reply]
Ok yes I remember this problem. I'm not 100% sure why the offset doesn't match the transition, however I'm quite sure that the current implementation was the only one that did not break the (spring) animation completely.
Therefore, I will not investigate this, as the animation of the BottomSheet is currently working more or less and I'm currently quite busy. However, if you want to look into this and open a pull request (that obviously doesn't break the (spring) Animations on iOS 13+) I will happily merge it!

Greetings,
Lucas
